services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: datetime.date,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        date = datetime.strptime(session_date, "%Y-%m-%d")
+        return MovieSession.objects.filter(show_time__date=date).all()
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
```

Comment:
Consider using `MovieSession.objects.filter(id=movie_session_id).first()` or handling the `MovieSession.DoesNotExist` exception to avoid potential errors if the movie session with the given ID does not exist.